test/matrix_generator.cc

```diff
@@ -24,7 +24,6 @@
 
 // -----------------------------------------------------------------------------
 const int64_t idist_rand  = 1;
-const int64_t idist_rands = 2;
```

Comment:
Instead of removing this, I suggest we change the enums below to reference it. E.g.,
```
 enum class TestMatrixType {
-    rand      = 1,  // maps to larnv idist
-    rands     = 2,  // maps to larnv idist
-    randn     = 3,  // maps to larnv idist
+    rand  = idist_rand,   // 1
+    rands = idist_rands,  // 2
+    randn = idist_randn,  // 3
     randb,
     zero,
     one,
@@ -52,9 +52,9 @@ enum class TestMatrixType {
 };
 
 enum class TestMatrixDist {
-    rand      = 1,  // maps to larnv idist
-    rands     = 2,  // maps to larnv idist
-    randn     = 3,  // maps to larnv idist
+    rand  = idist_rand,   // 1
+    rands = idist_rands,  // 2
+    randn = idist_randn,  // 3
```

Reply:
After discussion, I added a commit for this.